bluesky_kafka/tests/conftest.py

```diff
@@ -49,7 +49,17 @@ def kafka_bootstrap_servers(request):
 
 
 @pytest.fixture(scope="function")
-def temporary_topics(kafka_bootstrap_servers):
+def test_broker_authorization_config():
```

Comment:
I don't think you want the word test in the name of the fixture.

Reply:
That's a good point.